src/components/Stepper/__tests__/__snapshots__/index.tsx.snap

```diff
@@ -486,22 +486,22 @@ exports[`Stepper should click on plus button with a step value 1`] = `
   margin: 0 4px;
 }
 
-.cache-1iz16l3[aria-disabled='true'] {
+.cache-1sfzzi6[aria-disabled='true'] {
   opacity: 0.5;
   cursor: not-allowed;
 }
 
-.cache-1iz16l3[aria-disabled='false'] {
+.cache-1sfzzi6[aria-disabled='false'] {
   cursor: pointer;
   touch-action: manipulation;
 }
 
-.cache-1iz16l3>.etyrq8m4 {
+.cache-1sfzzi6>.etyrq8m4 {
   fill: #4f0599;
 }
 
-.cache-1iz16l3:hover:not([disabled]) {
-  background: #eeeeff;
+.cache-1sfzzi6:hover:not([disabled]) {
+  background: #b2b6c3;
```

Comment:
@matthprost is neutral.backgroundHover dodgy?

Reply:
You can use `backgroundWeakDisabled` in you case also you should use primary instead of neutral as it was gray200 and gray200 === light violet